lib/xboxkrnl/xboxkrnl.h

```diff
@@ -2448,7 +2448,7 @@ XBAPI NTSTATUS NTAPI PhyInitialize
 
 /**
  * Read the status information from the NICs' registers
- * @param update
+ * @param update If FALSE, the kernel returns the cached value, otherwise the hardware is polled and the cached value updated
```

Comment:
nit: "returns a cached value"? It's never explained that this is cached before

Reply:
Reworded the function description.